src/__builtins.di

```diff
@@ -68,4 +68,14 @@ version (DigitalMars)
     alias __builtin_fabs  = core.stdc.math.fabs;
     alias __builtin_fabsf = core.stdc.math.fabsf;
     alias __builtin_fabsl = core.stdc.math.fabsl;
+
+    import core.bitop;
+
+    alias __builtin_bswap32 = core.bitop.bswap;
+    alias __builtin_bswap64 = core.bitop.bswap;
```

Comment:
Mapping both `__builtin_bswap32` and `__builtin_bswap64` to the same function doesn't look safe. Consider the following C code:

```C
#include <assert.h>

int main()
{
    int x = 123;
    long long bs32 = __builtin_bswap32(x);
    long long bs64 = __builtin_bswap64(x);
    assert(bs32 == 0x7b000000ll);
    assert(bs64 == 0x7b00000000000000ll);
    return 0;
}
```

And D implementation of this feature probably needs unit tests.

Reply:
They're D overloads.

Reply:
> They're D overloads.

Yes, they are. But this doesn't make them safe. In my C example above, the lines `long long bs32 = __builtin_bswap32(x);` and `long long bs64 = __builtin_bswap64(x);`have exactly the same argument type (int) and exactly the same return value type (long long). If you alias them to the same `bswap` function, then they will naturally return exactly the same result for the same input. But the result must be different, as confirmed by the assert lines. So in order to work correctly, the implementation of `__builtin_bswap64` needs to explicitly cast its argument to a 64-bit type.

Additionally please consider the following D code:
```D
import std.stdio, std.conv, core.bitop;

void main()
{
    int x = 123;

    writefln("%x", x.to!short.bswap);    // prints 7b000000
    writefln("%x", x.to!ushort.bswap);   // prints 7b000000

    writefln("%x", x.to!int.bswap);      // prints 7b000000
    writefln("%x", x.to!uint.bswap);     // prints 7b000000

    writefln("%x", bswap(cast(long)x));  // prints 7b000000
    writefln("%x", bswap(cast(ulong)x)); // prints 7b00000000000000

    writefln("%x", x.to!long.bswap);     // prints 7b00000000000000
    writefln("%x", x.to!ulong.bswap);    // prints 7b00000000000000
}
```
The printed results for `.to!short`, `.to!ushort` and `cast(long)` seem to be fishy. But this probably needs to be filed as a separate bug.

Reply:
It's not to the same function, but to the same template. Importing templates works in ImportC.